models/Product.py

```diff
@@ -24,7 +25,8 @@ class Item(EmbeddedDocument):
 class Product(Document):
     """Product Schema"""
 
-    _id = ObjectIdField(primary_key=True)
+    _id = ObjectIdField(default=ObjectId(), primary_key=True)
+    user_id = ObjectIdField(db_field="User", required=True)
```

Comment:
Not needed as it'll never be used since `user_id` is already in the group model(`organizer_id`) and the group model also has the `product_id`.